docs/timeline.md

```diff
@@ -58,21 +58,35 @@ When this target is missed, you may experience UI jank or dropped frames.
 See [Flutter performance profiling](https://flutter.dev/docs/testing/ui-performance) for more detailed information on
 how to analyze your app's performance.
 
-## Flame Chart
+## Frame Flame Chart
 
-The flame chart shows the event trace for a single frame. The top-most event spawns the event below it, and so on and so
+The frame flame chart shows the event trace for a single frame. The top-most event spawns the event below it, and so on and so
 forth. The UI and GPU events are separate event flows, but they share a common timeline (displayed at the top of the
 flame chart). This timeline is strictly for the given frame. It does not reflect the clock shared by all frames.
 
-<img src="images/timeline_flame_chart.png" width="800" />
+<img src="images/timeline_frame_flame_chart.png" width="800" />
 
 The flame chart supports zooming and panning. Scroll up and down to zoom in and out, respectively. To pan around, you
-can either click and drag the chart or scroll horizontally. You can also click an event to view more details in the
-details section below the chart.
+can either click and drag the chart or scroll horizontally. You can also click an event to view CPU profiling information
+in the section below the chart.
 
-## Event Details
+## CPU Profiler
```

Comment:
maybe add the word preview to this heading?

Reply:
done.